pkgs/stdenv/darwin/make-bootstrap-tools.nix

```diff
@@ -1,14 +1,20 @@
-{ pkgspath ? ../../.., test-pkgspath ? pkgspath, system ? builtins.currentSystem, crossSystem ? null }:
+{ pkgspath ? ../../.., test-pkgspath ? pkgspath
+, system ? builtins.currentSystem, crossSystem ? null, bootstrapFiles ? null
+}:
+
+let cross = if crossSystem != null
+      then { inherit crossSystem; }
+      else {};
+    custom-bootstrap = if bootstrapFiles != null
```

Comment:
Is this just a convenience to test the stdenv, etc. using the new bootstrap-tools without having to edit `pkgs/stdenv/darwin/default.nix`?

Reply:
Yep, as I described [here](https://github.com/NixOS/nixpkgs/pull/121055#issuecomment-852429669).